test/mocks/server/mocks.h

```diff
@@ -27,7 +27,9 @@ namespace Server {
 
 class MockOptions : public Options {
 public:
-  MockOptions();
+  // If it is not provided the configPath will be an empty string in the mock.
```

Comment:
nit: s/it/path (though I think this is kind of obvious and comment can probably be removed)